JavaScript/JavaScript1/Week4/voice-assistant.js

```diff
@@ -0,0 +1,73 @@
+let userName = "";
+let todos = [];
+
+function getReply(command) {
+    const lowerCommand = command.toLowerCase();
```

Comment:
Good idea to clean the user input with `toLowerCase`. You can go further and use `trim()` too.